supplements/monsters-of-the-multiverse/race-genasi.xml

```diff
@@ -0,0 +1,432 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<elements>
+	<info>
+		<name>Genasi from Monsters of the Multiverse</name>
+		<update version="0.0.1">
+			<file name="race-genasi.xml" url="https://raw.githubusercontent.com/AuroraLegacy/elements/master/supplements/monsters-of-the-multiverse/race-genasi.xml" />
+		</update>
+	</info>
+
+	<element name="Genasi" type="Race" source="Monsters of the Multiverse" id="ID_WOTC_MOTM_RACE_GENASI">
+		<description>
+			<p>Tracing their ancestry to the genies of the Elemental Planes, each genasi can tap into the power of one of the elements. Air, earth, fire, and water-these are the four pillars of the Material Plane and the four types of genasi. Some genasi are direct descendants of a genie, while others were born to non-genasi parents who lived near a place suffused by a genie's magic.</p>
+			<p class="indent">A typical genasi has a life span of 120 years. </p>
+			<h4>Genasi Traits</h4>
+			<p>As an genasi, you have the following racial traits.</p>
+			<p class="indent"><strong><em>Creature Type.</em></strong> You are a Humanoid.</p>
+			<p class="indent"><strong><em>Size.</em></strong> You are Medium or Small. You choose the size when you select this race.</p>
+			<p class="indent"><strong><em>Speed.</em></strong> Your walking speed is 30 feet.</p>
+			<p class="indent"><strong><em>Darkvision.</em></strong> You can see in dim light within 60 feet of you as if it were bright light, and in darkness as if it were dim light. You discern colors in that darkness only as shades of gray.</p>
+			<p class="indent"><strong><em>Subrace.</em></strong> Four major subraces of genasi are found among the worlds of D&amp;D: air genasi, earth genasi, fire genasi, and water genasi. Choose one of these subraces.</p>
+		</description>
+		<sheet display="false" />
+		<rules>
+			<select type="Ability Score Improvement" name="Ability Score Improvement Option (Genasi)" supports="Custom Ability Score Improvement Option" />
+			<select type="Size" name="Size (Genasi)" supports="ID_SIZE_SMALL||ID_SIZE_MEDIUM" />
```

Comment:
needs to be type="Racial Trait"

Reply:
Fixed in [9b0b613](https://github.com/AuroraLegacy/elements/pull/119/commits/9b0b613d832291909c6d2fd9d553b65b82637b5e)